src/fabric_view_changes.erl

```diff
@@ -461,6 +464,13 @@ validate_start_seq(DbName, Seq) ->
             {error, {bad_request, Reason}}
     end.
 
+get_changes_epoch() ->
+    {ok, Epoch} = application:get_env(fabric, changes_epoch),
+    Epoch.
+
+increment_changes_epoch() ->
+    application:set_env(fabric, changes_epoch, get_changes_epoch() + 1).
```

Comment:
yes, true, I just really didn't like writing it to config (and requiring operators to mentally increment the value). will ponder.